addons/web/static/src/views/kanban/kanban_header.xml

```diff
@@ -8,7 +8,7 @@
                     <t t-esc="groupName"></t>
                     <span class="badge text-bg-500 rounded-pill lh-1 ms-2" t-esc="group.count"></span>
                 </div>
-                <span t-if="!env.isSmall and !group.isFolded"
+                <span t-if="!group.isFolded"
```

Comment:
... but this will re-introduce the issue with `social`, no?

Reply:
Nope, double checked :)

Reply:
Could you you (fetch first...) and rebase your branch? I can't test on runbot and the branch is red